mlir/docs/Dialects/ArmSME.md

```diff
@@ -1,13 +1,28 @@
 # 'ArmSME' Dialect
 
-[TOC]
+Basic dialect to target Arm SME.
+
+This dialect defines custom and LLVM IR intrinsic operations that are used to
+target Arm Scalable Matrix Extension. Through the available lowerings one can,
```

Comment:
Feel free to ignore, I know it's correct, but saying 'one' always always sounds odd to me :P

```suggestion
target Arm Scalable Matrix Extension. Through the available lowerings you can,
```